Packs/GoogleCloudFunctions/Integrations/GoogleCloudFunctions/GoogleCloudFunctions.py

```diff
@@ -169,15 +171,38 @@ def format_parameters(parameters: str) -> str:
     return json.dumps(dict(pairs))
 
 
+def set_default_region(region):
```

Comment:
I would rename this function to `get_...` or `resolve_...`. It is not setting anything.

Reply:
Done